specification/sql/resource-manager/readme.md

```diff
@@ -115,10 +115,9 @@ input-file:
 - Microsoft.Sql/preview/2019-06-01-preview/ServerAzureADAdministrators.json
 - Microsoft.Sql/preview/2019-06-01-preview/syncGroups.json
 - Microsoft.Sql/preview/2019-06-01-preview/syncMembers.json
-- Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json
```

Comment:
We just move FailoverManagedInstance to ManagedInstance.json file. Actually, there is no change about API. The operation group name of FailoverManagedInstance is ManagedInstance. Thus, It should be in  ManagedInstance json file.

The reason is that previous developers manually put FailoverManagedInstance API into FailoverManagedInstance.json file. They did not follow the spec file naming rules.